packages/block-editor/src/components/use-block-commands/index.js

```diff
@@ -229,6 +176,21 @@ const useActionsCommands = () => {
 		return { isLoading: false, commands: [] };
 	}
 
+	const rootClientId = getBlockRootClientId( clientIds[ 0 ] );
+	const canInsertDefaultBlock = canInsertBlockType(
+		getDefaultBlockName(),
+		rootClientId
+	);
+	const canDuplicate = blocks.every( ( block ) => {
+		return (
+			!! block &&
+			hasBlockSupport( block.name, 'multiple', true ) &&
+			canInsertBlockType( block.name, rootClientId )
+		);
+	} );
+	const canRemove = canRemoveBlocks( clientIds, rootClientId );
+	const canMove = canMoveBlocks( clientIds, rootClientId );
```

Comment:
I think we also need to check that there is more than one block, just as [the Move to action on the block toolbar does](https://github.com/WordPress/gutenberg/blob/3a7404dd80067715da65caffb7f830b3ca218617/packages/block-editor/src/components/block-settings-menu-controls/index.js#L93).